pybliometrics/scopus/tests/test_SubjectClassifications.py

```diff
@@ -20,6 +20,15 @@
                               fields=['description', 'detail'], refresh=30)
 
 
+def test_module():
```

Comment:
I think one of them is sufficient. The additional assertions do not test different functionality.